src/org/tensorflow/yolo/view/ClassifierActivity.java

```diff
@@ -136,11 +184,65 @@ private void renderAdditionalInformation(final Canvas canvas) {
             }
         }
 
-        lines.add("Frame: " + previewWidth + "x" + previewHeight);
-        lines.add("View: " + canvas.getWidth() + "x" + canvas.getHeight());
-        lines.add("Rotation: " + sensorOrientation);
-        lines.add("Inference time: " + lastProcessingTimeMs + "ms");
+// shimatani
+        // lines.add("Frame: " + previewWidth + "x" + previewHeight);
+        // lines.add("View: " + canvas.getWidth() + "x" + canvas.getHeight());
+        // lines.add("Rotation: " + sensorOrientation);
+        // lines.add("Inference time: " + lastProcessingTimeMs + "ms");
+
+        Log.d(LOGGING_TAG, String.format("検出名: %s", nowRecognizedClass));
```

Comment:
Please convert kanji chars to English ones.

Reply:
```ClassifierActivity.java
Log.d(LOGGING_TAG, String.format("Detected object name: %s", nowRecognizedClass));
```